x-pack/solutions/observability/plugins/apm/public/components/app/onboarding/instructions/otel_agent.tsx

```diff
@@ -253,7 +239,10 @@ export function OpenTelemetryInstructions({
                 target="_blank"
                 href="https://github.com/open-telemetry/opentelemetry-specification/blob/v1.10.0/specification/protocol/exporter.md"
```

Comment:
I know it was already there, but could you update these links with an elastic link, please?

Reply:
You mean the URL shortner or there is an elastic repo with similar docs?